HoloLensCommander/HoloLensCommander/Dialogs/MobileCenterAppsDialogViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
```

Comment:
Currently, this project's only dependency is on WindowsDevicePortalWrapper. Is the json data from the app center complex enough to warrant using the Netwtonsoft component? If not, I would prefer we not take an additional dependency.

Reply:
I can help with the deserialization code if we can avoid the component,.

Reply:
@davidkline-ms  Yes, there are only 2 instances of deserializing the JSON response data, and it’s a relatively simple object (~15 fields total, but I only use 4). Help with this would be awesome so that we do not add more dependencies.

Reply:
Cool. Since you only use the four, we should be able to only define the subset when configuring the object for the built in json serializer

Reply:
@davidkline-ms what's the easiest way to write our own json parser for these objects? Are you thinking we parse the strings ourselves? Or are there built in functions I am not aware of?

Reply:
here is a sample class definition from the WindowsDevicePortalWrapper project:

         using System.Runtime.Serialization;

        [DataContract]
        public class BatteryState
        {
            [DataMember(Name = "AcOnline")]
            public bool IsOnAcPower { get; private set; }

            [DataMember(Name = "BatteryPresent")]
            public bool IsBatteryPresent { get; private set; }

            [DataMember(Name = "Charging")]
            public bool IsCharging { get; private set; }

            [DataMember(Name = "DefaultAlert1")]
            public int DefaultAlert1 { get; private set; }

            [DataMember(Name = "DefaultAlert2")]
            public int DefaultAlert2 { get; private set; }

            [DataMember(Name = "EstimatedTime")]
            public uint EstimatedTimeRaw { get; private set; }

            [DataMember(Name = "MaximumCapacity")]
            public int MaximumCapacity { get; private set; }

            [DataMember(Name = "RemainingCapacity")]
            public int RemainingCapacity { get; private set; }
        }

Reply:
This file shows how WDPW validates and handles the JSON.

Reply:
In the BatteryState example, if all you cared about was whether or not a battery existed and if the system was plugged in, you could include only the first two members of the class.